File: NetSample/VBEventHandler/Form1.vb
Public Class Form1
    '也可以支接使用 Handle 加上method, 來連結click Event
    Private Sub btnAddEvent_Click(sender As Object, e As EventArgs) Handles btnAddEvent.Click
        AddHandler btnOri.Click, AddressOf btnOri_Click
    End Sub
    '要傳入兩個參數 sender:object (就是button)   , e:EventArg
    Private Sub btnOri_Click(sender As Object, e As EventArgs)
        MessageBox.Show("已加上Event了")
    End Sub
End Class

File: NetSample/CSharpAddEvent/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CSharpAddEvent
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        //增加事件
        private void btnAddEvent_Click(object sender, EventArgs e)
        {
            btnOri.Click += btnOri_Click;
        }
        //移除事件
        private void btnRemoveEvent_Click(object sender, EventArgs e)
        {
            btnOri.Click -= btnOri_Click;
        }
        //btnOri事件執行的Method
        private void btnOri_Click(object sender, EventArgs e)
        {
            MessageBox.Show("已加入事件");
        }
    }
}
